app/views/return-requirements/returns-how-do-you-want.njk

```diff
@@ -0,0 +1,25 @@
+{% extends 'layout.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set rootLink = "/licences/" + licenceId %}
```

Comment:
you need to have `/system/` at the beginning of the url

Reply:
thanks, updated